code/game/objects/items/weapons/manuals.dm

```diff
@@ -1367,4 +1367,84 @@
 				Remember; once you have been afflicted with brain trauma you are over four times as likely to suffer it again! Keep your patient's best interests at heart.
 				</body>
 			</html>
-			"}
\ No newline at end of file
+			"}
+
+/obj/item/weapon/book/manual/brainwashing

```

Comment:
What does brainwashing have to do with pKAs?

Reply:
It was leftover from an poorly done devmerge on my part.